addons/project/models/project_task_recurrence.py

```diff
@@ -102,8 +82,9 @@ def _create_next_occurrence_values(self, occurrence_from):
             for field, value in fields_to_postpone.items()
         })
 
+        create_values['priority'] = '0'
         create_values['stage_id'] = occurrence_from.project_id.type_ids[0].id if occurrence_from.project_id.type_ids else occurrence_from.stage_id.id
         create_values['child_ids'] = [
-            Command.create(self._create_next_occurrence_values(child)) for child in occurrence_from.with_context(active_test=False).child_ids
+            child.with_context(copy_project=True).sudo().copy(self._create_next_occurrence_values(child)).id for child in occurrence_from.with_context(active_test=False).child_ids
```

Comment:
I don't really like this line, since you're handling the child hierarchy here instead of letting the copy() method do it. But on the other hand, since every child has potentially custom data compared to its original version, I don't really have a better solution to offer. :man_shrugging:

Reply:
Me neither. 😕